### PR TITLE
update test and integration to use company.name

### PR DESCRIPTION
**Pull request by ros3bud**
@cgrayson, this one too. LGTY?

### CODE REVIEW

**Comment by cgrayson** [comment]
To change a field that's been deprecated, the old field (`company`) should either be removed completely, or else still supported by tests and marked as deprecated in the code. In either case, some change to line 313 is needed.

File: src/mortgage.coffee [hunk 1]
```diff
@@ -36,7 +36,7 @@ request = (vars) ->
     borrower_comments: lead.comments
 
     # employment
-    employer_name: lead.company
+    employer_name: lead.company?.name
```
Comment (cgrayson) [inline src/mortgage.coffee]: If `company` support is being kept for some transition period, this should use the value from `company.name` primarily, but fall back to `company` if it's empty.

**Comment by ros3bud** [comment]
@cgrayson, removed the company field. Is this alright?

**Comment by cgrayson** [comment]
LGTM

**Pull request merged**